<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicForm</class>
 <widget class="QWidget" name="MusicForm">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>420</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="QWidget" name="mainContent" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>771</width>
     <height>381</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QListWidget" name="fileList">
    <property name="geometry">
     <rect>
      <x>355</x>
      <y>11</y>
      <width>361</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="folderList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>341</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
  </widget>
  <zorder>mainContent</zorder>
  <zorder>backButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
